### DISCUSSION TITLE
Hydraulics ECAM page

### DISCUSSION THREAD

**Discussion by Flycat62** [flybywiresim/aircraft#5590] [Help] [answered]
Why I can't see HYDR ECAM page (black screen) when I push HYDR button? I have the newest version of (v0.6.2) of a32nx after a clean installation. Please help!!!!

**Comment by hotshotp** [comment, marked as answer]
because hydraulics are not implemented in stable and only dev

**Reply by Flycat62** [reply]
Thanks for your reply.
